README.md

```diff
@@ -12,10 +12,21 @@ To install the requirements first run the command
 ```
 sudo pip install -r requirements.txt
 ```
+There is no `sudo` in Windows. Hence, you should run the command:
+
+```
+
+pip install -r requirements.txt
+
+```
+
+
```

Comment:
hey so these newlines dont do anything to the preview can you remove those except maybe 1?